packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -523,7 +523,6 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
       throw new Error(`wallet does not support ${this.getDisplayName()}`)
     }
 
-    await this.assertSwitchChain(wallet)
```

Comment:
Included in here to avoid merging release into develop, we need this anyway

Reply:
Not so sane as it will be merged into the squashed commit of this PR but well, for the sake of simplicity, let's go